torch/_ops.py

```diff
@@ -1097,10 +1103,12 @@ def __getattr__(self, key):
             setattr(self, key, overload)
             self._dir.append(key)
             return overload
-        except RuntimeError:
-            raise AttributeError(
+        except RuntimeError as e:
+            # TEMPORARY, WILL REMOVE.
```

Comment:
This needs updating before merging right?

Reply:
yep, just wanted to flush out all the other places we throw at runtime in C++

Reply:
looks like this did not fail any tests in CI, which is pretty cool :D